pkg/apis/cr/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -2,7 +2,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2023 The Kanister Authors.
+Copyright The Kubernetes Authors.
```

Comment:
Is that intentional? Do we want to change the copyright to Kubernetes?

Reply:
It came by running `make codegen`. This is by default from package when we re regenerate.

Reply:
Fixed it by adding `boilerplate.go.txt` in repo.